### PR TITLE
Various updates

### PR DESCRIPTION
**Pull request by jeromekelleher**
- Tweaks to mutation generation appendix (@petrelharp, what do you think?)
- Apply @mmatschiner's comments
- Apply @eldonb's comments

### CODE REVIEW

File: paper.tex [hunk 1]
```diff
@@ -1736,18 +1744,23 @@ \section*{Mutation generation}
...
-Any more efficient tree-by-tree algorithm would have to make use of the fact
-that structure is shared between adjacent trees---effectively, therefore,
-working with shared edges.
+Since each edge is processed independently, the algorithm
+scales linearly with the number of edges in the tree sequence.
+
+% jk - I don't think this para helps to be honest - we've already made the
+% argument above about tree-by-tree algorithms, and talking about an
+% efficient version where we use the diffs is just going to confuse matters
+% I think.
```
Comment (petrelharp) [inline paper.tex:1753]: sounds good.

File: paper.tex [hunk 2]
```diff
@@ -1810,7 +1823,8 @@ \section*{Mutation generation}
 simulating with the \texttt{HKY} mutation model will result in silent mutations
 if not all equilibrium frequencies are the same.
 The presence of silent mutations may at first be surprising
-(although they don't affect calculations of statistics in any way),
+(although they are fully accounted for in downstream calculations of statistics
+by \tskit),
```
Comment (petrelharp) [inline paper.tex:1827]: "accounted for" isn't the same as "doesn't affect" - how about "although they do not affect downstream statistics of genetic diversity as calculated by \tskit"?

**Review by petrelharp — APPROVED** [review]
LGTM! Minor comment.

File: paper.tex (hunk 2 above)
Comment (jeromekelleher) [reply paper.tex:1827]: I was worried that people would interpret "they don't affect calculations" as "tskit doesn't realise they are there and does something stupid". We want to say that "tskit does exactly the right thing, even with silent mutations". See what I mean?

Comment (petrelharp) [reply paper.tex:1827]: gah, yes, hm:
"although the value of statistics computed downstream by tskit are not changed by the presence of silent mutations"
or
"although statistics computed downstream by tskit are not affected by the presence of these silent mutations"

Comment (jeromekelleher) [reply paper.tex]: I addressed this in 77924995276eb858efbab0c56bca5ca4f753e404 by adding a short paragraph at the end - I think it's too important a point to try to get across in the middle of a sentence. Please do follow up if you'd like to make changes, though.

Comment (petrelharp) [reply paper.tex]: looks great.

**Pull request merged**